test/ejabberd_tests/tests/last_SUITE.erl

```diff
@@ -61,9 +61,10 @@ end_per_testcase(CaseName, Config) ->
 %%--------------------------------------------------------------------
 last_online_user(Config) ->
     escalus:story(Config, [{alice, 1}, {bob, 1}],
-                  fun(Alice, _Bob) ->
+                  fun(Alice, Bob) ->
                           %% Alice asks about Bob's last activity
-                          escalus_client:send(Alice, escalus_stanza:last_activity(bob)),
+                          BobShortJID = escalus_client:short_jid(Bob),
```

Comment:
What's the rationale of deprecating previous version? The new one adds verbosity to code.